core/test/services/i18n.spec.js

```diff
@@ -0,0 +1,91 @@
+const chai = require('chai');
```

Comment:
Here my proposal on how we could refactor these tests: [i18n-modified.spec.js.zip](https://github.com/SAP/luigi/files/3392769/i18n-modified.spec.js.zip). Please apply it as you consider more appropriate and let me know if you have any question ;)

Reply:
Done.